doc/UserGuide.tex

```diff
@@ -1379,6 +1380,16 @@ \section{Reading, Writing and Formatting}
 The \texttt{xml} option will produce an XML-version of the development graph
 for our change management broker.
 
+The \texttt{db} option will save the development graph to a database.
+This can be either an SQLite database or a PostgreSQL database (only available
+in the \texttt{hets-server} package). For an SQLite database, you need to pass
+the parameter \texttt{--database-file=FILEPATH} to specify the database file to
+save to. If the file does not yet exist, \Hets will create it.
+For a PostgreSQL database, you need to pass the
+\texttt{--database-config=FILEPATH} and \texttt{--database-subconfig=KEY}
+parameters that point to a Ruby-on-Rails-compatible \texttt{database.yml}
+configuration file.
```

Comment:
could you please include a sample database.yml file, or point to the sample file on github?

Reply:
Done with 6b71baeef. I linked it to the master branch, hoping that it will remain there.